aura/api_repository.py

```diff
@@ -235,6 +237,14 @@ def make_api_call_and_wait_for_data_api_status(
     instance_id = data["aura_instance"]["id"]
     status = data.get("status")
 
+    if method == "POST":
+        api_key = data["api_key"]
```

Comment:
Not all POST requests for data apis return an api_key (e.g. if someone is using a jwks endpoint for auth instead). My python is very rusty but wouldn't this result in an error if api_key is undefined?

Reply:
Fair, trying to be too fancy, changed it to just check if `api_key` is in the data.